portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,32 @@
+
+//this function generates a random quote to display, also making sure that the previously displayed random image is not the same as the current random image.
 function randomizeQuote() {
```

Comment:
nit: for comments the Google style is to start with a space and a capitol letter (as you would with a sentence).

// This function generates... 

Please update all comments in this class to match this style.

Reply:
Yes, done updating it in all files.